### ISSUE TITLE
Pokemon Ultra Sun/Ultra Moon - Broken textures

### ISSUE THREAD

**Issue by Hexagon12** [citra-emu/citra#3133]
Tested on Nightly c810a3f.

The broken textures appear next to the entrance to Route 2 in Hau'oli City.
The log doesn't tell anything specific, except `Unhandled texture type 5`.
Log file: 
[log.txt](https://github.com/citra-emu/citra/files/1485472/log.txt)

Citra screenshot: 
![citra-qt_2017-11-19_12-34-10](https://user-images.githubusercontent.com/25362478/32989936-7b580138-cd28-11e7-874f-07285a3e432e.png)

Screenshot, which was taken from my 3DS:
![top_0001](https://user-images.githubusercontent.com/25362478/32989947-b6d40090-cd28-11e7-9784-6a5e5f56c554.png)

**Comment by CDAGaming** [comment]
Can confirm this, saw it quite a bit on some paths

**Comment by BreadFish64** [comment]
Reproduced with Nvidia GPU, does not occur on Intel integrated.

**Comment by pPkMnh4to8994h** [comment]
same thing appears on SM, on same place.

**Comment by ddnava** [comment]
Does the texture flicker when the camera moves? This looks like some weird Z-fighting

**Comment by BreadFish64** [comment]
@ddnava yes, it appears to be z-fighting, the effect only occurs on some GPUs

**Comment by spixi** [comment]
A little off-topic, but I cannot confirm the sentence “require quite powerful hardware to achieve higher frame rates” mentioned in the [wiki](https://citra-emu.org/game/pokemon-ultra-sun-ultra-moon/). It works (until the Photo Club crash) fine on

Intel Core i7-4800MQ
NVIDIA GeForce GT 730M
Linux 4.4.39
Citra master-ddee442f

However it fails on the integrated Intel GPU

**Comment by BreadFish64** [comment]
Is say the 4800mq is decently powerful

**Comment by spixi** [comment]
But the GeForce GT 730M is mediocre (~ 30 FPS). And it still works better than the integrated GPU in the CPU (~ 2 FPS). However, only the GeForce has the texture problem.

**Comment by BreadFish64** [comment]
@spixi GPU is nearly irrelevant, my GT 740m barely goes above 40% usage, but this is getting off topic

**Comment by Hexagon12** [comment]
@spixi "master-ddee442f" Yeah, that's not even a official build. Keep in mind that the performance report is based on Nightly only.

**Comment by Reuuz** [comment]
where the download button?

**Comment by tgsm** [comment]
https://github.com/citra-emu/citra-nightly/releases/tag/nightly-516

**Comment by wwylele** [comment]
clean up off-topic

**Comment by CJGreninja** [comment]
how do I download this game

**Comment by LOuroboros** [comment]
I was about to submit a new issue, but I guess I should just mention it here.
I think that I found this same issue in Route 9, the route right before Konikoni City.
I'm using Citra Canary 1451.
![citra-qt-2019-08-17_16-37-27](https://user-images.githubusercontent.com/4485172/63216612-2d95b680-c10e-11e9-8f3d-2b80897ed99a.png)
![citra-qt-2019-08-17_16-39-17](https://user-images.githubusercontent.com/4485172/63216613-2d95b680-c10e-11e9-9344-4b80c482fab1.png)

**CPU**: i3 2120
**RAM**: 4GB (2x2 running on single channel because reasons)
**GPU**: Gigabyte NVIDIA GeForce GT 1030 2GB GDDR5, Driver Version 431.60
**OS**: Windows 7 Professional SP1 x64

**Comment by Jacalz** [comment]
I am also running in to the issues mentioned here.

**Comment by LOuroboros** [comment]
This issue is still present on the latest Canary release.
![Untitled462](https://user-images.githubusercontent.com/4485172/96798538-7bbfdd00-13d7-11eb-9fcb-54fe7202397f.png)
![Untitled464](https://user-images.githubusercontent.com/4485172/96798542-7d89a080-13d7-11eb-83ee-5310679f30ee.png)
![Untitled467](https://user-images.githubusercontent.com/4485172/96798717-d6f1cf80-13d7-11eb-92fe-9545ddcb26b6.png)

Specs:
**CPU**: i3 2120
**RAM**: 4GB *(2x2 running on single channel because reasons)*
**GPU**: Gigabyte NVIDIA GeForce GT 1030 2GB GDDR5, Driver Version 456.38
**OS**: Windows 10 Education Version 2004 *(x64)*

**Comment by LOuroboros** [comment]
This issue is still present on the latest Canary release *(1928)*. I don't think I checked it before, but it doesn't look like it's an upscaling related issue since setting the IR to Native doesn't solve it.
![1](https://user-images.githubusercontent.com/4485172/106341087-c7a5dc00-627a-11eb-9951-8140e091c9b7.png)
![2](https://user-images.githubusercontent.com/4485172/106341090-c8d70900-627a-11eb-9e31-0160098f93ce.png)

Specs:
**CPU**: i3 2120
**RAM**: 4GB *(2x2 running on single channel because reasons)*
**GPU**: Gigabyte NVIDIA GeForce GT 1030 2GB GDDR5, Driver Version 456.38
**OS**: Windows 10 Education Version 2004 *(x64)*

**Comment by RianCaio** [comment]
hi i know its a old issue but it can be fixed by using scaleforce filter its not a proper fix but better than nothing.

**Comment by poudink** [comment]
While ScaleForce does indeed fix the Z-Fighting at IRs above 1x, it also unfortunately causes visual artifacts (see the black lines around the trees). The visual artifacts are also visible with every filter except xBRZ (and of course none), so you have to choose between the Z-Fighting and the artifacts.
![USUMSF](https://user-images.githubusercontent.com/29778535/118069513-5645f100-b372-11eb-9706-c2d4654bd0f0.png)

**Comment by LOuroboros** [comment]
Still happening in Canary 2013.

![citra-qt_3201](https://user-images.githubusercontent.com/4485172/148341202-c5668868-d1eb-4467-820c-08a8c206b46c.png)

**Comment by LumiRev** [comment]
Still happening in Nightly 1950.
![image](https://github.com/citra-emu/citra/assets/53419914/a0d02678-7c08-4851-99f9-4650bac2e3fa)

**Comment by GPUCode** [comment]
Does this issue occur with vulkan?

**Comment by Lolothepro** [comment]
> Does this issue occur with vulkan?

yes

**Comment by Lolothepro** [comment]
https://github.com/citra-emu/citra/assets/68275908/314c656d-4f27-4dfc-83b8-79b9d13fa4d0

**Comment by LOuroboros** [comment]
> Does this issue occur with vulkan?

Yup.

![citra-qt_20230916_021507666](https://github.com/citra-emu/citra/assets/4485172/bf986417-5588-4125-883e-2727fcbb18ac)

**Comment by Steveice10** [comment]
Can reproduce on Windows with latest NVIDIA drivers, cannot reproduce on Mac with MoltenVK.
